infrastructure/iam_for_lambda.tf

```diff
@@ -0,0 +1,35 @@
+resource "aws_iam_role" "nfish-des-role-lambda_s3_access" {
+  name = "nfish-des-role-lambda_s3_access"
+
+  assume_role_policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": "sts:AssumeRole",
+      "Principal": {
+        "Service": "lambda.amazonaws.com"
+      },
+      "Effect": "Allow",
+      "Sid": ""
+    }
+  ]
+}
+EOF
+}
+
+resource "aws_iam_role_policy_attachment" "nfish-des-lambda_s3_access" {
+  role       = aws_iam_role.nfish-des-role-lambda_s3_access.name
+  for_each = toset([
+  "arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole",
+  "arn:aws:iam::aws:policy/AmazonDynamoDBFullAccess",
```

Comment:
To assign full access to a resource is bad practice

Try to specify the actions and resources.
Is also a good practice

Reply:
ok